bin/setup_services.sh

```diff
@@ -24,8 +24,12 @@ set -euo pipefail
 /app/socorro-cmd gcs create "${TELEMETRY_GCS_BUCKET}"
 
 # Delete and create Elasticsearch indices
-/app/socorro-cmd es delete
-/app/socorro-cmd es create
+/app/socorro-cmd legacy_es delete
+/app/socorro-cmd legacy_es create
+if [ "${ELASTICSEARCH_MODE^^}" == "PREFER_NEW" ]; then
```

Comment:
This has `set -u` set, so this errors:

```
/app/bin/setup_services.sh: line 29: ELASTICSEARCH_MODE: unbound variable
make: *** [Makefile:68: setup] Error 1
```

```suggestion
ELASTICSEARCH_MODE="${ELASTICSEARCH_MODE:-LEGACY_ONLY}"
if [ "${ELASTICSEARCH_MODE^^}" == "PREFER_NEW" ]; then
```